Documentation/ExtensionInstallation/Index.rst

```diff
@@ -15,6 +15,14 @@ Extension Installation
 
 .. _install-extension-without-composer:
```

Comment:
`.. _install-extension-without-composer:` defines a label for the header "Install an Extension Without Composer". The label must be directly followed by the header (with a blank line in between). The new text must be put before the label. 

See https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Hyperlinks.html#link-targets